cmake-modules/AzureDoxygen.cmake

```diff
@@ -38,6 +38,8 @@ function(generate_documentation PROJECT_NAME PROJECT_VERSION)
         # Use MathJax instead of latex to render formulas
         set(DOXYGEN_USE_MATHJAX YES)
 
+        set(DOXYGEN_REPEAT_BRIEF NO)
```

Comment:
I couldn't easily find what `DOXYGEN_REPEAT_BRIEF` does, I imagine you already know this Victor, can you share, and maybe add it as a comment?

Reply:
Oh... is this that allows us to not have the extra empty new line after `brief`? Interesting.

Reply:
> Oh... is this that allows us to not have the extra empty new line after brief? Interesting.

Not really. It is hard to explain by writing here. I will try, but, if it is not clear, we can have a call.

Whenever DOXYGEN_REPEAT_BRIEF is YES (default value), For a class like

```
/***
* @brief something
*/
class Something { ..  };
```

The generated documentation in the HTML page will be like

```
class Something  (Name of the class)

something brief (brief from docs)

...
...
...

Detail
something brief (brief from docs is mentioned here again)

The docs after the brief.
```

So, when turning DOXYGEN_REPEAT_BRIEF to NO, we won't see a detail section that repeats the brief when we only have a brief in the docs.  But in the case we have more docs next to the brief like:

```
/**
* @brief I am a brief
*  I am just the next thing after the brief. You can call me details.
* /
```

In this case, the generated docs are:

 ```
class Something  (Name of the class)

I am a brief

...
...
...

Detail
 I am just the next thing after the brief. You can call me details.

```

The `More` link depends directly on this.  
 -  If the docs contains only a brief, there will not be a More link because there will not be a details section.
 -  If the docs contains a brief and also more things after it (or even just an empty line after the brief), then there will be a `more` link that takes you to the details section to see the info after the brief (the brief is not repeated here because of DOXYGEN_REPEAT_BRIEF)